README.md

```diff
@@ -1,37 +1,117 @@
+
+| Command                                         | Description                                                      |
+|-------------------------------------------------|------------------------------------------------------------------|
+| `sm2 --start CUSTOMS_FINANCIALS_ALL`            | Runs all dependencies                                            |
+| `sm2 --start CUSTOMS_EMAIL_FRONTEND_ALL `       | Runs all email dependencies                                      |
+| `sm2 -s`                                        | Shows running services                                           |
+| `sm2 --stop CUSTOMS_FINANCIALS_EMAIL_THROTTLER` | Stop the micro service                                           |
+| `sbt run` or `sbt "run 9872"`                   | (from root dir) to compile the current service with your changes |
+
+
+### Runtime Dependencies
+(These are subject to change and may not include every dependency)
 
 * `AUTH`
 * `AUTH_LOGIN_STUB`
 * `AUTH_LOGIN_API`
+* `BAS_GATEWAY`
+* `CA_FRONTEND`
+* `SSO`
 * `USER_DETAILS`
 * `EMAIL`
 * `CONTACT_FRONTEND`
 
-You can use the CUSTOMS_EMAIL_THROTTLER profile in service manager to start these services.
+### Login enrolments
+
+The service can be accessed by using below enrolments and with below sample EORI numbers, via http://localhost:9949/auth-login-stub/gg-sign-in (on local) or https://<host:port>/auth-login-stub/gg-sign-in on DEV/QA/STAGING
+
+Redirect URL - `/customs/payment-records`
+
+| Enrolment Key  | Identifier Name | Identifier Value |
+|----------------|-----------------|------------------|
+| `HMRC-CUS-ORG` | `EORINumber`    | `GB744638982000` |
+| `HMRC-CUS-ORG` | `EORINumber`    | `GB744638982001` |
+
+## Testing
+
+The minimum requirement for test coverage is 90%. Builds will fail when the project drops below this threshold.
+
+### Unit Tests
+
+| Command                                | Description                  |
+|----------------------------------------|------------------------------|
+| `sbt test`                             | Runs unit tests locally      |
+| `sbt "test/testOnly *TEST_FILE_NAME*"` | runs tests for a single file |
+
+### Coverage
+
+| Command                                  | Description                                                                                                 |
+|------------------------------------------|-------------------------------------------------------------------------------------------------------------|
+| `sbt clean coverage test coverageReport` | Generates a unit test coverage report that you can find here target/scala-3.3.5/scoverage-report/index.html |
+
+## Available Routes
+
+You can find a list of microservice specific routes here - `/conf/app.routes`
+
+Application entrypoint:  `/customs/payment-records`
 
-Once these services are running, you should be able to do `sbt "run 9872"` to start in `DEV` mode.
+## Feature Switches
```

Comment:
I think better to add just 'Not applicable' under Feature Switches section (and remove other details) as its better to explicitly mention that there are no feature flags for this service